main.py

```diff
@@ -37,32 +37,28 @@
 BILLING_ACTIVATED = CONFIG['billing']['activate'].get(bool)
 DUMP_JSON_FILE_NAME = CONFIG['dump_json_file_name'].get()
 
-DEBUG_ACTIVE_PROJECTS = CONFIG['debug']['active_projects'].get(bool)
 DEBUG_ENRICHED_PROJECTS = CONFIG['debug']['enriched_projects'].get(bool)
 DEBUG_FILTERED_BY_PROJECTS = CONFIG['debug']['filtered_by_projects'].get(bool)
 DEBUG_FILTERED_BY_USERS = CONFIG['debug']['filtered_by_users'].get(bool)
 DEBUG_FILTERED_BY_AGE = CONFIG['debug']['filtered_by_age'].get(bool)
 DEBUG_GROUPED_BY_OWNERS = CONFIG['debug']['grouped_by_owners'].get(bool)
+DEBUG_FILTERED_BY_ORGS = CONFIG['debug']['filtered_by_org'].get(bool)
 
 def main():
     client = _get_resource_manager_client()
 
     logger.info('Retrieving Projects.')
-    projects = _get_projects(client)
-
-    logger.info('Filtering active Projects.')
-    active_projects = list(filter(filter_active_projects_matching_org_level(
-        ORGS_FILTER), projects))
-
-    if DEBUG_ACTIVE_PROJECTS:
-        logger.debug('Active Projects:\n%s', pformat(active_projects))
+    active_projects = _get_projects(client)
 
     logger.info('Calculating Project age information.')
     enriched_projects = _enrich_project_info_with_age(active_projects)
 
     logger.info('Retrieving Project owners information.')
     enriched_projects = _enrich_project_info_with_owners(client, enriched_projects)
 
+    logger.info('Retrieving Project organization information.')
+    enriched_projects = _enrich_project_info_with_orgs(client, enriched_projects)
```

Comment:
Because this is an "expensive" operation that will make a lot of API calls, I think we should make it optional via a config flag. Similar to `BILLING_ACTIVATED` below.

Reply:
- now it is optional to enrich projects with org info, similar to BILLING, 14a6234